src/components/views/rooms/EventTile.js

```diff
@@ -745,14 +745,26 @@ export default class EventTile extends React.Component {
         }
 
         if (this.props.mxEvent.sender && avatarSize) {
-            avatar = (
+            if(this.props.mxEvent.getType()==='m.room.third_party_invite') {
```

Comment:
This should be `getContent().third_party_invite` like in https://github.com/matrix-org/matrix-react-sdk/blob/b10f7a47fdc3497f5d15901ca21898dfc5c66bbf/src/TextForEvent.js#L33-L42

Reply:
Hey @t3chguy! Thanks for reviewing my PR. I'm not sure if I understood what exactly you wanted me to change to `getContent().third_party_invite`. Please have a look and let me know if I got it right. Thanks a ton! :D

Reply:
Looks good other than some small stylistic nits, I'll try test this soon!